benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -44,6 +44,30 @@
     "PageID": "1.0",
     "PageLabel": "Handle Further Evidence"
   },
+  {
+    "CaseEventID": "actionFurtherEvidence",
+    "CaseFieldID": "showPostponementDetailsPage",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "21/08/2021",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "2.0",
+    "PageLabel": "Hearing request party",
+    "PageShowCondition": "showPostponementDetailsPage=\"Yes\"",
+    "FieldShowCondition": "postponementRequestDetails=\"DoNotShow\""
+  },
+  {
+    "CaseEventID": "actionFurtherEvidence",
+    "CaseFieldID": "postponementRequestDetails",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "21/08/2021",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "2.0",
+    "PageLabel": "Handle Further Evidence"
```

Comment:
Sorry to be pedantic, but the page label is different from the one above even though they are on same page. Suggest you just remove the page label for this one to avoid confusion.